### PR TITLE
image-builder: improve ova built stability

### PR DESCRIPTION
**Pull request by jaxesn**
*Issue #, if available:*

*Description of changes:*

- Use pause.tar to pull in previous version pause image to make kubeadm init happy instead of pulling from ecr which will sometimes timeout on ssl handshake
- Use crictl sha256 file that now exists instead of generating it to avoid crictl building while image-builder and breaking checksum validation
- Fix goss validation due to change in version on kube binaries in latest eks-d releases

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/168#" title="Author self-approved">jaxesn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh
Comment (vignesh-goutham) [inline projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh:85]: I want to make sure if this pause.tar is an OCI image tarball. I have a faint memory of this being one of the issue why we pulled from ECR as opposed to tarball.

Comment (jaxesn) [reply projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh:85]: it is, i tested this.

Its created [here](https://github.com/aws/eks-distro/blob/main/Common.mk#L69) with output_type being oci.

**Review by vignesh-goutham — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/168#" title="Author self-approved">jaxesn</a>*, *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/168#pullrequestreview-805290299" title="LGTM">vignesh-goutham</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [jaxesn,vignesh-goutham]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
